packages/contracts-bedrock/src/L2/OptimismSuperchainERC20.sol

```diff
@@ -48,16 +42,10 @@ contract OptimismSuperchainERC20 is IOptimismSuperchainERC20Extension, ERC20, IS
     struct OptimismSuperchainERC20Metadata {
```

Comment:
Do we really need this full struct now?

Reply:
I leave it to keep the ERC7201 implementation, but maybe we can use a slot only for the `remoteToken` and it would be enough